eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -49,40 +49,80 @@ param(
   [string]$PRBody = $PRTitle,
 
   [Parameter(Mandatory = $false)]
-  [string]$PRLabels
+  [string]$PRLabels,
+
+  [Parameter(Mandatory = $false)]
+  [string]$UserReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$TeamReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$Assignees
 )
 
+. "${PSScriptRoot}\logging.ps1"
```

Comment:
Why not import common.ps1?

Reply:
So far we only need the logging logic here, so I did not feel the need to import common.

Reply:
OK. I'm fine with this for now but at some point we might want to consider always importing common.ps1 just to avoid any differences for scripts we use in our eng-system. The only time we should avoid it is if we want to limit our dependencies in a script to use it in other contexts.